### PR TITLE
Refs #12846: bump hammer_cli_katello to 0.0.19

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by komidore64** [comment]
apj

File: rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -28,8 +28,8 @@ Requires:  %{?scl_prefix_ruby}ruby(release)
 Requires:  %{?scl_prefix_ruby}ruby(abi)
 %endif
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.1.3
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 1.0.0
```
Comment (johnpmitsch) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:32]: I could be missing something, but it looks like your greater than and less than are switched.

Comment (johnpmitsch) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec:32]: oh nvm I see `1.0.0` not `0.1.0` now :flushed:

File: rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 Summary: Katello command plugin for the Hammer CLI
 Name:    %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.0.18
+Version: 0.0.19
 Release: 3%{?dist}
```
Comment (komidore64) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: don't you want to change the release back to `1`?

Comment (ehelms) [inline rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Good catch!

**Comment by ehelms** [comment]
Udated

**Pull request merged**